pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -86,6 +86,30 @@ type RequestedToCapacityRatioArgs struct {
 	Resources []ResourceSpec
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourcesLeastAllocatedArgs holds arguments used to configure NodeResourcesLeastAllocated plugin.
+type NodeResourcesLeastAllocatedArgs struct {
+	metav1.TypeMeta
+
+	// Resources to be managed, if no resource is provided, default resource set with both
+	// the weight of "cpu" and "memory" set to "1" will be applied.
+	// Resource with "0" weight will not accountable for the final score.
```

Comment:
What about unlisted?

Reply:
I didn't get it the "unlisted" you mean here. :(

Reply:
It might be obvious actually. But I was referring to default resources (like memory) not being listed.

Let me suggest alternative wording:

```
Resources to be considered when scoring.
The default resource set includes "cpu" and "memory" with an equal weight.
Allowed weights go from 1 to 100.
```

Then we should just disallow 0 weight (it doesn't make sense, you should be removing the item from the list instead).

Reply:
agree... I'd expect us to treat 0 and negative weights as invalid config and prevent their use, not silently ignore the resources

Reply:
The suggested word is applied, negative weight is banned in the plugins along with this push, as to "0" I think it's harmless to ignore it unless I missed something.  I will investigate to something like `jsonschema validator` to validate the configuration of yaml in the long term, the static check will make the validation easier.